pytri/__init__.py

```diff
@@ -39,63 +39,104 @@ class pytri:
     .
     """
 
-    def __init__(self):
+    def __init__(self, **kwargs):
         """
         Create a new visualizer frame.
 
         Arguments:
-            None
+            figsize (int, int): A 2-tuple of pixel sizes. Either may be None
+                to auto-rescale in that axis.
+            width (int): The width to set the figure
+            height (int): The height to set the figure
 
         """
+        self.debug = kwargs.get('debug', False)
         scripts = [
-            "https://threejs.org/examples/js/controls/TrackballControls.js",
+            # None listed.
```

Comment:
Do we expect to still need this? Maybe mention explicit use case if so.

Reply:
I want to leave it in because it was a pain to figure out the first time. I'll add docs and explain. We may also want to someday use this as a way to push backward-compatible substrate patches (SUBPDATES!) even to older versions of pytri.